velox/core/SimpleFunctionMetadata.h

```diff
@@ -26,6 +26,9 @@
 #include "velox/type/Type.h"
 #include "velox/type/Variant.h"
 
+inline constexpr char kPrecisionVariable[] = "a_precision";
+inline constexpr char kScaleVariable[] = "a_scale";
```

Comment:
static is required to prevent external linkage and is the common pattern used.
`static constexpr const char*`